CMakeLists.txt

```diff
@@ -103,6 +103,7 @@ endif ()
 ################################################################################
 
 # needed for target doc and fulldoc
+find_program( NOSETESTS NAMES nosetests )
```

Comment:
Very nice!
Can you please remove the comment above as that seems outdated and not very helpful anyway.